.sphinx/_static/furo_colors.css

```diff
@@ -68,6 +71,8 @@ body {
             --color-background-hover: #666;
             --color-brand-primary: #fff;
             --color-brand-content: #06C;
+            --color-api-background: #333;
+            --color-api-background-hover: #666;
```

Comment:
A little concerned about the contrast for links with this one - but it's only on hover, so should be okay for now.